src/components/timeline/TimelineViewControls.svelte

```diff
@@ -216,15 +211,7 @@
 
   function scrollToSelection() {
     const time = getSelectionTime();
-    if (!isNaN(time) && (time < viewTimeRange.start || time > viewTimeRange.end)) {
-      const midSpan = time + getIntervalInMs($selectedSpan?.duration) / 2;
-      const start = Math.max(maxTimeRange.start, midSpan - viewDuration / 2);
-      const end = Math.min(maxTimeRange.end, midSpan + viewDuration / 2);
-      dispatch('viewTimeRangeChanged', {
```

Comment:
Do we no longer need to dispatch this event to properly set the Timeline?

Reply:
@duranb i moved it to the plan store, so both this and the activity directives table uses the plan store to throw this event. This is used in the "Auto scroll to selected activity" - but let me know if I broke something!